WordPress/src/main/res/layout/post_cardview.xml

```diff
@@ -69,12 +69,23 @@
             tools:text="text_status"
             tools:visibility="visible" />
 
-        <FrameLayout
+        <LinearLayout
```

Comment:
Just a tip, not necessary for this review: you can hit Opt+Cmd+L to reformat the layout file.